fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -146,11 +138,6 @@
     private boolean dropJavaBatchSchema = false;
     private boolean grantJavaBatchSchema = false;
 
-    // By default, the dryRun option is OFF, and FALSE
-    // When overridden, it simulates the actions.
-    @SuppressWarnings("unused")
-    private boolean dryRun = false;
-
```

Comment:
I removed it as it was never used

Reply:
sounds good, though I'd still like to add it back in at some point!